server/webhook_worker.go

```diff
@@ -80,3 +83,44 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 
 	return nil
 }
+
+func (wh *webhook) FetchReporterNotification(bb []byte, reporter *jira.User) error {
+	if wh.eventTypes.ContainsAny("event_created_comment") || wh.eventTypes.ContainsAny("event_updated_comment") {
```

Comment:
```suggestion
	if wh.eventTypes.ContainsAny("event_created_comment", "event_updated_comment") {
```

Reply:
According to https://community-daily.mattermost.com/core/pl/97cg76sa4bgf5raq53dat1hrwo, we actually don't want to send these notifications for comment updates. Can you please ensure we are not sending comment update notifications for mentions or assignees either?